src/services/IPagoPAClientFactory.ts

```diff
@@ -7,5 +7,12 @@ export interface IPagoPAClientFactoryInterface {
   /**
    * Retrieves a configured instance of the API client.
    */
-  readonly getClient: () => ReturnType<PagoPAClient>;
+  readonly getClient: (
+    environment: PagoPAEnvironment
+  ) => ReturnType<PagoPAClient>;
+}
+
+export enum PagoPAEnvironment {
+  PRODUCTION = "production",
```

Comment:
keep values same as keys

```suggestion
  PRODUCTION = "PRODUCTION",
  TEST = "TEST"
```

Reply:
changed in #b235f21